mysql-test/t/mysqldump_gtid.inc

```diff
@@ -0,0 +1,69 @@
+--source include/have_log_bin.inc
+--source include/have_innodb.inc
+--source include/not_embedded.inc
+--source include/have_debug_sync.inc
+
+--echo #################################################
+--echo # Testing mysqldump with options: $MYSQLDUMP_OPTS
+--echo #################################################
+
+--let sleep_sec = 5
+
+--connect(con1, localhost, root)
```

Comment:
Add `count_sessions.inc` / `wait_count_sessions.inc`

Reply:
Not addressed

Reply:
done

Reply:
done